client/src/main/scripts/etc/slipstream-orchestrator.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=SlipStream Orchestrator Executor
+After=syslog.target
```

Comment:
Maybe we can replace it with:
```ini
[Unit]
Description=SlipStream Orchestrator Executor
Wants=network-online.target
After=syslog.target network.target network-online.target
```

This comment also apply to the file `slipstream-node.service`

You can find some details about waiting on the network access here http://unix.stackexchange.com/questions/126009/cause-a-script-to-execute-after-networking-has-started